static/js/components/PrivacyTab.js

```diff
@@ -30,10 +30,9 @@ class PrivacyTab extends ProfileFormFields {
             We care about your privacy.
           </Cell>
         </Grid>
-        <Grid className="profile-tab-grid">
+        <Grid className="profile-tab-grid privacy-form">
           <Cell col={12}>
-            <h4>Who can see your profile?</h4>
-            { this.boundRadioGroupField(['account_privacy'], '', this.privacyOptions) } <br />
+            <PrivacyForm {...this.props} />
```

Comment:
Here you have `PrivacyForm` inside of a `<Cell col={12}>` component, this makes the `<Grid>` and `<Cell>` components declared in the render method of `PrivacyForm` unnecessary - if you were getting the warning about 'adjacent JSX elements' a simple solution is to wrap the `<h4>` element and the call to `boundRadioGroupField` in a `<div>`.